cmd/zed/zed.d/zed-functions.sh

```diff
@@ -261,6 +261,13 @@ zed_notify_email()
         return 1
     fi
 
+    # no subject passed as option?
+    if ! [ "${ZED_EMAIL_OPTS}" != "${ZED_EMAIL_OPTS/@SUBJECT@/}" ] ; then
```

Comment:
Could you git rid of the double negation and just do?:
```
    if  [ "${ZED_EMAIL_OPTS}" == "${ZED_EMAIL_OPTS/@SUBJECT@/}" ] ; then
```

Reply:
Done.